src/new-client/src/App/Trades/TradesGrid/components/ComparatorSelect.tsx

```diff
@@ -2,15 +2,15 @@ import { useRef } from "react"
 import styled from "styled-components"
 import { usePopUpMenu } from "@/utils"
 import type { NumFilterContent } from "../../TradesState"
-import { DateFilterContent } from "../../TradesState/filterState/dateFilterState"
+import { DateFilterContent } from "@/App/Trades/TradesState/filterState"
```

Comment:
```suggestion
import { DateFilterContent } from "../../TradesState/filterState"
```

Let's use the relative import for internals.